Dockerfile

```diff
@@ -27,8 +27,8 @@ RUN pip install poetry==$POETRY_VERSION && \
 FROM registry.access.redhat.com/ubi9/ubi-minimal
 SHELL ["/bin/bash", "-c"]
 
-ARG PYTHON_VERSION=3.10.11
-ARG CONDA_VERSION=23.3.1
+ARG PYTHON_VERSION=3.10.12
```

Comment:
Can we update python version - 3.11.6?

Reply:
Good question! That would require a bit more testing than a minor version bump like this was. Out of curiosity, outside of it being better to be on a later version, are there particular things missing since it's a `3.10` base?

Reply:
yes, actually version incompatible with mlflow.pyfunc because in mlflow using 3.11.6

Reply:
@ask664: No update, unfortunately. I've raised this with others.

You could try editing the Dockerfile [1] to bump it to `3.11.x` and build an image in the meantime.

[1] https://github.com/SeldonIO/MLServer/blob/master/Dockerfile#L1